invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -73,7 +73,8 @@ <h2 class="ui header">{{ community.metadata.title }}</h2>
     </div>
     {% set header_menu_items = {
       'search': (_('Search'), url_for('invenio_communities.communities_detail', pid_value=community.id), 'search icon', True),
-      'members': (_('Members'), url_for('invenio_communities.members', pid_value=community.id), 'users icon', true),
+      'members': (_('Members'), url_for('invenio_communities.members', pid_value=community.id), 'users icon', True),
+      'requests': (_('Requests'), url_for('invenio_communities.communities_requests', pid_value=community.id), 'file outline icon', permissions.can_create),
```

Comment:
```suggestion
      'requests': (_('Requests'), url_for('invenio_communities.communities_requests', pid_value=community.id), 'comments outline icon', permissions.can_create),
```
what about comments icon instead to mark requests ?